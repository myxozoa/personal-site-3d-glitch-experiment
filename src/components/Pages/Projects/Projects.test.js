import React from 'react';
import { shallow } from 'enzyme';

import Component from './index';

describe('<Projects />', () => {
  it('renders correctly', () => {
    const component = shallow(<Component />);

    expect(component).toMatchSnapshot();
  });
});