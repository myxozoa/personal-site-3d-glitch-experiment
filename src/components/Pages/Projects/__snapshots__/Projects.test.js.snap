// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Projects /> renders correctly 1`] = `
<div
  className="container"
>
  <h2
    className="glitch"
    data-text="projects"
  >
    projects
  </h2>
  <section
    className="projects"
  >
    <a
      className="link"
      href="https://www.maincourse.app"
      key="fdsh09978"
    >
      <div
        className="project"
      >
        <div
          className="projectName"
        >
          main course
        </div>
        <div
          className="projectDesc"
        >
          restaurant app for the ages
        </div>
      </div>
    </a>
    <a
      className="link"
      href="https://github.com/myxozoa/LPM"
      key="dhans9770"
    >
      <div
        className="project"
      >
        <div
          className="projectName"
        >
          lpm
        </div>
        <div
          className="projectDesc"
        >
          project manager assistant
        </div>
      </div>
    </a>
    <a
      className="link"
      href="https://www.maincourse.app"
      key="fdsh09978"
    >
      <div
        className="project"
      >
        <div
          className="projectName"
        >
          main course
        </div>
        <div
          className="projectDesc"
        >
          restaurant app for the ages
        </div>
      </div>
    </a>
    <a
      className="link"
      href="https://github.com/myxozoa/LPM"
      key="dhans9770"
    >
      <div
        className="project"
      >
        <div
          className="projectName"
        >
          lpm
        </div>
        <div
          className="projectDesc"
        >
          project manager assistant
        </div>
      </div>
    </a>
    <a
      className="link"
      href="https://www.maincourse.app"
      key="fdsh09978"
    >
      <div
        className="project"
      >
        <div
          className="projectName"
        >
          main course
        </div>
        <div
          className="projectDesc"
        >
          restaurant app for the ages
        </div>
      </div>
    </a>
    <a
      className="link"
      href="https://github.com/myxozoa/LPM"
      key="dhans9770"
    >
      <div
        className="project"
      >
        <div
          className="projectName"
        >
          lpm
        </div>
        <div
          className="projectDesc"
        >
          project manager assistant
        </div>
      </div>
    </a>
  </section>
</div>
`;
